tests/behat/features/plugin-check.feature

```diff
@@ -29,3 +29,73 @@ Feature: Test that the WP-CLI command works.
       """
       All output should be run through an escaping function
       """
+
+    When I run `wp plugin check hello.php --format=csv`
```

Comment:
```suggestion
    When I run the WP-CLI command `wp plugin check hello.php --format=csv`
```

Should mention `the WP-CLI command` as we did for other cases. What do you think @swissspidy?

🔢 Apply to other places

Reply:
Yes indeed, good point! That's required for actually being able to collect the code coverage.

Reply:
PR updated.

Reply:
Earlier `wp` is omitted when `When I run WP-CLI command` was used like. I have also followed similar. Hope that is ok. `When I try the WP-CLI command 'plugin check foo-bar'`